openfecwebapp/templates/legal-search-results.html

```diff
@@ -2,7 +2,7 @@
 {% import 'macros/legal-search.html' as legal_search %}
 
 {% block title %}
-  Search results for "{{query}}"
+  Search results for “{{query}}”
```

Comment:
It looks like these are smart double-quotes (or smart-style double-quotes) - is this an intended change or should these still be plain text double-quotes?